doc/release/README.bugs.rst

```diff
@@ -18,15 +18,16 @@ in confidence by the Chapel team, please mail it to:
 Please include as much information as possible in your email.  For
```

Comment:
I think we should go head and split it out. The changes all look good to me, so you can put me down as a reviewer.